src/main/java/svc/exceptions/NotFoundException.java

```diff
@@ -0,0 +1,35 @@
+package svc.exceptions;
+
+public class NotFoundException extends Exception {
```

Comment:
You should be able to just extend `RuntimeException` here instead of `Exception`.